src/index.js

```diff
@@ -667,7 +670,7 @@ see doc: https://www.rabbitmq.com/reliability.html#consumer-side`);
                     consumerDebug('Handler success');
                     // otherwise internal subscribe (rpc…)
                     if (qualifier) {
-                        config.transport.info(`${rpc ? '◀ ' : '◁ '} ${qualifier}`, {
+                        transport.info(`${rpc ? '◀ ' : '◁ '} ${qualifier}`, {
```

Comment:
why is it only this one log that takes the specific transport into account, and not the other logs?

Reply:
Missed the error log, fixed ✅